src/components/shared/RoomIconPicker.tsx

```
@@ -0,0 +1,286 @@
+          const config = ROOM_ICONS[iconKey];
+          if (!config) return null;
+
+          const IconComponent = config.icon;
+          const isSelected = selectedIcon === iconKey;
+          const label = t(config.labelKey, iconKey);
+
+          return (
+            <button
+              key={iconKey}
+              type="button"
+              role="radio"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<input type="radio">

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)